### ISSUE TITLE
Instability is a lie?

### ISSUE THREAD

**Issue by anoobindisguise** [CleverRaven/Cataclysm-DDA#58667]
### Describe the bug

Instability doesn't seem to be working correctly. In theory you are supposed to get 90% positive mutations and 10% neutral mutations until 8 mutations total, at which point you start getting a 5% greater chance of negative mutations and a 5% lower chance of getting a positive mutation. You can theoretically get a bad mutation if you roll a positive mutation such as Intestinal Fortitude that wants you to get a negative mutation first (weak stomach/nausea/vomitous) and not all positive-value mutations are "good" but you are supposed to not get negative mutations right away and even then they should be rather rare at 1/20, 2/20, 3/20... (you have a 58% chance of not getting ANY bad mutations after a total of 12 mutations for ~400 instability.)

In theory.
I just got a negative mutation (Weakening) as my first mutation on an Alpha mutant-to-be, I should have *zero* instability and according to my debug log, I really did have zero instability. But that's not what happens in practice. 

### Steps to reproduce

1. Attempt to mutate with appropriate vitamins.
2. Observe negative mutations appearing much too soon.

### Expected behavior

Negative mutations should only start happening at 9+ with an increasing 5% chance of a negative mutation each mutation beyond 8 assuming no Robust Genetics is reducing your instability.

### Screenshots

![image](https://user-images.githubusercontent.com/56016372/175454776-d842d5a8-8915-401a-a568-90e20ff76111.png)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.22000.739 (21H2)
- Game Version: e34eeaa [64-bit]
- Graphics Version: Tiles
- Game Language: System language []
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions],
    Bionic Slots [cbm_slots]
]

**Comment by mkrutov** [comment]
To confirm that you need statistics. Try to mutate 10 times with 0 instability, see results.

Single unlucky roll doesn't mean its broken.

**Comment by Amoebka** [comment]
There shouldn't even be the possibility of an unlucky roll to begin with. According to the documentation, the chance of a negative mutation that isn't a prereq to a positive one at 0 instability should be 0% flat.

**Comment by Terrorforge** [comment]
Duplicate of #57716.

There's a tidying up of the mutation systems being implemented in #57912 that may or may not be relevant, but it seems to currently be on hold for personal reasons.

**Comment by Stadler76** [comment]
Quoting myself from Discord here;
Instability as it is now is a must have, if you intend to get all the pros and cons of a mutation category aka all mutations of a single category. And unintentionally a waste of mutagen, if you got all bad mutations of said category.
e. g. try to get to the end of the black dragon mutation without getting spammed, that you need a more exotic mutagen...
I did so during my tests ... and gave up. And I'm aware, that black dragon is not vanilla. I just think it's a good example.

In other words: I'd say, that this issue is not fixed and should be reopened.
